rust/lib/src/lib.rs

```diff
@@ -51,19 +33,43 @@ pub fn init_new(server_uri: String, data_dir: String) -> String {
 
     seed
 }
+fn construct_uri_load_config(
+    uri: String,
+    data_dir: String,
+) -> Result<(zingoconfig::ZingoConfig, u64), String> {
+    let server = construct_server_uri(Some(uri));
 
-pub fn init_from_seed(server_uri: String, seed: String, birthday: u64, data_dir: String) -> String {
-    let server = construct_server_uri(Some(server_uri));
-
-    let (mut config, _latest_block_height) = match zingolib::load_clientconfig(server, None) {
+    let (mut config, latest_block_height) = match zingolib::load_clientconfig(server, None) {
         Ok((c, h)) => (c, h),
         Err(e) => {
-            return format!("Error: {}", e);
+            return Err(format!("Config load Error: {}", e));
```

Comment:
In zingo-mobile code base exists a convention about how to catch errors in the responses of RPCModule... In most of the places the App is waiting an error when starts with "Error:", can you change this line to: "Error Config load......"?????? Thanks.

Reply:
Like:   `"Error: Config load:..."` ?

Reply:
OK, I pushed an update.  How does that look?

Reply:
Yeap... thanks.